lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -554,6 +554,16 @@ defmodule ExUnit.Assertions do
       {name, meta, context}, acc when is_atom(name) and is_atom(context) ->
         {:ok, [{name, [generated: true] ++ meta, context} | acc]}
 
+      node = {:"<<>>", meta, context}, acc when is_list(context) ->
+        if Enum.all?(context, &is_binary/1) do
+          vars = Enum.map(context, fn(var) ->
+            {String.to_atom(var), [generated: true] ++ meta, nil}
```

Comment:
Not sure if `nil` is correct here, should I be using `__CALLER__` or something else here?